lib/mumukit/bridge/runner/response_type.rb

```diff
@@ -54,12 +54,28 @@ def build_hash(response)
       end
     end
 
+    class Mixed < Structured
+      def build_hash(response)
+        super(response).merge response_type: :mixed, status: response['exit'].to_sym
```

Comment:
Why are we only using the exit state instead of combining the results of both the tests results AND the general `exit`?

Since this response types combines both structured tests and unestructured output, I'd expect it to fail or fail with warnings if there are failed tests or failed expectation results.

Reply:
I let the runner decide the final status here https://github.com/mumuki/mumuki-html-runner/blob/b8c37806d25b76b6b535c8032426de0ad28a158e/lib/test_hook.rb#L28

Do you think it should be bridge's responsability?

Reply:
I think that logic is quite generic -  `is_dom_ok && are_scripts_ok ? :passed : :failed` it looks like a custom scenario of the more general rule of `is_output_ok && are_tests_ok ? :passed : :failed `, isn't it?